packages/dashboard/src/components/resourceExplorer/index.test.tsx

```diff
@@ -1,55 +0,0 @@
-import * as React from 'react';
```

Comment:
Deleting this as it's noisy and (with the tested component unused right now) not currently adding value. Before re-adding the tested component it should be broken down into more testable constituent parts.